client/packages/common/src/intl/locales/en/common.json

```diff
@@ -2087,5 +2087,6 @@
   "warning.check-before-vaccinating": "Vaccination records from other stores may not be up to date. Ensure you have synced recently, and always confirm against the patient's physical vaccination card first.",
   "warning.field-not-parsed": "{{field}} not parsed",
   "warning.nothing-to-supply": "Nothing left to supply!",
+  "label.invoice-received": "Received",
```

Comment:
I thought invoice-received might be clearer for the meaning.
But I'm actually now wondering about `invoice-status-received` or something would be better?

Reply:
Ok, but I think we should probably refactor them all to better names. I think not having proper context is bad for translations...